app/Main.hs

```diff
@@ -25,7 +25,7 @@ url :: BaseUrl
 url = BaseUrl Http "127.0.0.1" 8200 ""
 
 token :: VaultToken
-token = VaultToken "s.AnHJuzOxRiD12eLxALFytccC"
```

Comment:
This will just introduce fuss into the commit history the token should be redacted anyway (not that it's valid... hopefully) but it might give some people false "hope" and might trigger those "secret finding bots" that apparently exist.

Reply:
Done